grammar.js

```diff
@@ -178,7 +179,7 @@ function hex_color_seq()  {
 }
 
 function snake_case_seq() {
-  return sep1(/[0-9a-z]+/, "_");
+  return sep1(token.immediate(word), token.immediate("_"));
```

Comment:
This idea makes sense. Though I think it should be `*_case_seq(...)` that should implement this, because applying `token.immediate` on the first rule would require these identifiers to be immediate as a whole. Does that makes sense?

```suggestion
  return seq(word, repeat(seq(token.immediate("_"), token.immediate(word)));
```

Reply:
whoops! Yeah this makes a little more sense now